test/noticed_test.rb

```diff
@@ -167,4 +172,12 @@ class Noticed::Test < ActiveSupport::TestCase
       With5MinutesDelay.new.deliver(user)
     end
   end
+
+  test "database delivery is executed without a delay" do
```

Comment:
If we do the validation, we can have a test to catch the `ArgumentError` raised instead.

Reply:
I'll also move it to the `test/delivery_methods/database_test.rb` file, as it makes more sense to be there

Reply:
👍